docs/usage.md

```diff
@@ -0,0 +1,187 @@
+
+## Subscribe to location updates
+
+Subscribe to `locationUpdate` events in order to display where users are within a space, such as which slide number they are currently viewing, or which cell or component they have selected. `locationUpdate` events for a space are sent when a user calls `locations.set()`, which can be configured to fire for whichever UI interactions you'd like. Use the `locations.on()` method to register a listener for `locationUpdates` events.
+
+The following is an example of subscribing to location updates for a space:
+
+```ts
+space.locations.on('locationUpdate', (update) => {
+  console.log(update);
+});
+```
+
+The following is an example `locationUpdate` event received by subscribers when a user changes location:
+
+```json
+{
+    "member": {
+    "clientId": "clemons@slides.com",
+    "isConnected": true,
+    "profileData": {
+      "username": "Claire Lemons",
+      "avatar": "https://slides-internal.com/users/clemons.png"
+    },
+    "location": {"slide": "3", "component": "slide-title"},
+    "lastEvent": { "name": "update", "timestamp": 1 },
+    "connections": ["2"],
+  },
+  "previousLocation": {"slide": "2", "component": null },
+  "currentLocation": {"slide": "3", "component": "slide-title"}
+}
+```
+
+The following are the properties of a `locationUpdate` event:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| member | The details of the user that has changed location. | [`spaceMember`](#subscribe-to-member-updates) |
+| previousLocation | The previous location of the user within the space. | any |
+| currentLocation | The current location of the user within the space. | any |
+
+To stop subscribing to location updates, users can call the `locations.off()` method.
+
+### Track an individual location or user
+
+It is also possible to track a specific location or user, rather than subscribing to all events using `createTracker()`.
+
+The following is an example of creating a tracker for a specific user based on their `clientId`, and then subscribing to updates for only that user:
+
+```ts
+const memberTracker = space.locations.createTracker(
+  (change) => change.member.clientId === 'clemons@slides.com'
+);
+
+memberTracker.on((change) => {
+  console.log(change);
+});
+```
+
+The following is an example of creating a tracker for a specific location, such as a single UI element, and then subscribing to updates for only that location:
+
+```ts
+const locationTracker = space.locations.createTracker(
+  (change) => change.previousLocation === 'slide-title'
+);
+
+locationTracker.on((change) => {
+  console.log(change);
+});
+```
+
+## Set a location
+
+Use `locations.set()` to publish a `locationUpdate` that will be received by all users subscribed to location updates.
+
+The following is an example of setting a location update: 
+
+```ts
+space.locations.set({slide: '3', component: 'slide-title'});
```

Comment:
Also worth clarifying at the beginning of the locations section that the location object is entirely user-defined. They can add whatever makes sense for their app. It could be a single level location or a multi-level (like which slide, then which component on it)